packages/react/src/evm/components/MediaRenderer.tsx

```diff
@@ -395,6 +405,44 @@ const LinkPlayer = React.forwardRef<HTMLAnchorElement, MediaRendererProps>(
 
 LinkPlayer.displayName = "LinkPlayer";
 
+const ModelViewer = React.forwardRef<HTMLCanvasElement, MediaRendererProps>(
```

Comment:
I wonder if there is a pre-made react-wrapper around the `model-viewer` library that we can just pull in instead?

Reply:
That was my initial thought as well, so I did a little research on this and was surprised to find no well-used and well-maintained projects in this category. Maybe I'm missing something, but here's what I had found:

This one only supports STL format:
https://www.npmjs.com/package/react-3d-model-component

These two are not currently maintained, and a brief attempt at implementing them didn't seem to work. I'm not sure if they are broken or if I'm missing something.
https://github.com/devhims/model-viewer-react
https://www.npmjs.com/package/react-model-viewer

This one I got to work fine with vanilla Javascript, but its React component doesn't seem to be implemented properly.
https://naver.github.io/egjs-view3d

This was a fairly brief search and brief attempts at using what already existed, but not finding any real clear winning solution is what prompted me to implement it the way I did. 

I'll dig a little deeper though, as I may very well have missed something. It's possible one of these can be made to work, or maybe there's something else that I've not found.

Reply:
sounds good this makes sense